spec/interactors/user/transfer_account_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+
+RSpec.describe User::TransferAccount do
+  subject { described_class.call(params) }
+
+  let!(:current_owner) { create(:user, :with_jwt) }
+
+  let(:params) do
+    {
+      current_owner: current_owner,
+      target_user_email: target_user_email,
+    }
+  end
+
+  context 'when the email address is valid' do
+    let(:target_user_email) { 'already@known.com' }
+
+    context 'when the target user already exists' do
+      let!(:target_user) { create(:user, email: target_user_email) }
+
+      it { is_expected.to be_a_success }
+
+      it 'gives the token ownership to the new user' do
+        transfered_jwt_ids = current_owner.jwt_api_entreprise_ids
+        subject
+
+        expect(target_user.jwt_api_entreprise_ids).to include(*transfered_jwt_ids)
+      end
+
+      it 'keeps the existing tokens of the target user' do
+        existing_tokens_id = target_user.jwt_api_entreprise_ids
+        subject
+
+        expect(target_user.jwt_api_entreprise_ids).to include(*existing_tokens_id)
+      end
+
+      it 'removes all token ownership from the previous user' do
+        subject
+
+        expect(current_owner.reload.jwt_api_entreprise).to be_empty
+      end
+
+      it 'notifies the new owner by email' do
+        expect(UserMailer).to receive(:transfer_ownership)
+          .with(current_owner, target_user)
+          .and_call_original
+
+        subject
+      end
+
+      it 'sets new tokens have been transfered' do
+        subject
+        target_user.reload
+
+        expect(target_user.tokens_newly_transfered).to eq(true)
+      end
+
+      it 'does not create a new user' do
+        expect { subject }.to_not change(User, :count)
+      end
+    end
+
+    context 'when the new owner does not exist in database' do
+      let(:target_user_email) { 'not@known.com' }
+
+      it { is_expected.to be_success }
+
+      it 'gives the token ownership to the new user' do
+        transfered_jwt_ids = current_owner.jwt_api_entreprise_ids
+        subject
+        target_user = User.find_by_email(target_user_email)
+
+        # added a sort call, equal was enforced but the order was not preserved in some
+        # test seeds
+        expect(target_user.jwt_api_entreprise_ids.sort).to eq(transfered_jwt_ids.sort)
```

Comment:
```suggestion
        expect(target_user.jwt_api_entreprise_ids).to contain_exactly(*transfered_jwt_ids)
```

un truc du genre (plutôt qu'un commentaire)